src/quacc/recipes/espresso/phonons.py

```diff
@@ -463,3 +449,106 @@ def _grid_phonon_subflow(
     )
 
     return _ph_recover_subflow(grid_results)
+
+
+@job
+def dvscf_q2r_job(
+    copy_files: SourceDirectorySchema,
+    parallel_info: dict[str] | None = None,
+    **calc_kwargs,
+) -> RunSchema:
+    """
+    Function to carry out a basic dvscf_q2r calculation. It should allow you to
```

Comment:
As a general comment, it would be helpful for users if you explain the purpose of the calculation itself. Many times throughout the modules, like here, it just says the name of the binary (e.g. `dvscf_q2r`). As a non-Espresso user, I don't know what this is, so I have no idea if the function is one I might want to use.